appinventor/appengine/src/com/google/appinventor/client/TopPanel.java

```diff
@@ -282,81 +190,5 @@ public void onClick(ClickEvent clickEvent) {
       Window.open(url, WINDOW_OPEN_LOCATION, WINDOW_OPEN_FEATURES);
     }
   }
-
-  private static class SignOutAction implements Command {
-    @Override
-    public void execute() {
-      // Maybe take a screenshot
-      Ode.getInstance().screenShotMaybe(new Runnable() {
-          @Override
-          public void run() {
-            Window.Location.replace(SIGNOUT_URL);
-          }
-        }, true);               // Wait for i/o
-    }
-  }
-
-  private static class DeleteAccountAction implements Command {
-    @Override
-    public void execute() {
-      if(ProjectListBox.getProjectListBox().getProjectList().getMyProjectsCount() > 0) {
-        Ode.getInstance().genericWarning(MESSAGES.warnHasProjects());
-      } else {
-        Ode.getInstance().getUserInfoService().deleteAccount(
-          new OdeAsyncCallback<String>(MESSAGES.accountDeletionFailed()) {
-            @Override
-            public void onSuccess(String delAccountUrl) {
-              if (delAccountUrl.equals("")) {
-                Ode.getInstance().genericWarning(MESSAGES.warnHasProjects());
-              } else {
-                if (delAccountUrl.equals("NONE")) {
-                  Window.Location.replace(SIGNOUT_URL);
-                } else {
-                  Window.Location.replace(delAccountUrl);
-                }
-              }
-            }
-          });
-      }
-      return;
-    }
-  }
```

Comment:
This needs to be moved to its own separate class and we need to restore the Delete Account behavior as noted before.